### PR TITLE
Add initial issuer.description tests.

### PR DESCRIPTION
**Pull request by aljones15**
1. Add invalid language object
2. Add direction
3. Add array of language tags

### CODE REVIEW

File: tests/10-vcdm2.js [hunk 1]
```diff
@@ -317,6 +317,13 @@ describe('Verifiable Credentials Data Model v2.0', function() {
         await assert.rejects(
           issue(require('./input/credential-issuer-object-no-id-fail.json')));
       });
+      it2('If present, the value of the "issuer.description" property ' +
+      'MUST be a string or a language value object as described in 10.1 ' +
+      'Language and Base Direction.', async function() {
+        await issue(require('./input/credential-issuer-description-ok.json'));
+        await issue(require(
+          './input/credential-issuer-description-language-en-ok.json'));
```
Comment (JSAssassin) [inline tests/10-vcdm2.js:325]: Do we have a test for issuer with no description property and it passes? If we don't we should add it here.

Comment (aljones15) [reply tests/10-vcdm2.js:325]: we have like 30-40 tests where the issuer has no description, but I can add those in a moment.

Comment (aljones15) [reply tests/10-vcdm2.js:356]: added tests for optional description: https://github.com/w3c/vc-data-model-2.0-test-suite/pull/48/commits/a131baa811052f3021ae262e5708b0f2b9917d03 and for optional name: https://github.com/w3c/vc-data-model-2.0-test-suite/pull/48/commits/cdcee49926fd4678e424eff70307ca23c039462f

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,8 @@
 
 ### Added
 - Add new tests for `issuer.name` including Internationalization settings.
-- A new test to ensure type urls are treated as a unordered set.
+- Add a new test to ensure type urls are treated as a unordered set.
```
Comment (JSAssassin) [inline CHANGELOG.md:7]:
```suggestion
- Add a new test to ensure type URLs are treated as an unordered set.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,7 +4,8 @@
...
+- Add new tests for `issuer.description` including Internationalization settings.
```
Comment (JSAssassin) [inline CHANGELOG.md:8]:
```suggestion
- Add new tests for `issuer.description` that includes internationalization settings.
```

**Review by JSAssassin — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (aljones15) [reply CHANGELOG.md:8]: I am not taking this suggestion as including is more active and precise than that includes.

**Pull request merged**